python/lsst/sims/utils/ModifiedJulianDate.py

```diff
@@ -0,0 +1,105 @@
+from lsst.sims.utils import taiFromUtc, utcFromTai, dutFromUtc
+from lsst.sims.utils import ut1FromUtc, utcFromUt1
+from lsst.sims.utils import dttFromUtc, ttFromTai, tdbFromTt
+
+__all__ = ["ModifiedJulianDate"]
+
+class ModifiedJulianDate(object):
+
+    def __init__(self, TAI=None, UTC=None):
+        """
+        Must specify either:
+
+        @param [in] TAI = the International Atomic Time as an MJD
+
+        or
+
+        @param [in] UTC = Universal Coordinate Time as an MJD
+        """
```

Comment:
Add to the documentation that this method (and all of the associated timeTransformation methods which use palpy.dat) will only accept single numbers. [is this something we could or should ask @timj to vectorize as well?]